src/App.tsx

```diff
@@ -8,32 +10,46 @@ function getRandomName(): string {
 }
 
 export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
-
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
-
-  // this code stops the timer
-  window.clearInterval(timerId);
+  const [clockName, setTimerName] = useState('Clock-0');
+  const [hasClock, setClockOn] = useState(true);
```

Comment:
```suggestion
  const [clockName, setClockName] = useState('Clock-0');
  const [hasClock, setHasClock] = useState(true);
```
Set state function should be named according to the noun variable